miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/integration/jira/JiraIssueManager.java

```diff
@@ -58,24 +67,39 @@
  */
 @Component
 public class JiraIssueManager implements IssueTrackerManager {
+
+  private static final DateFormat ISO8601_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+  private static final List<Character> SPECIAL_CHARS = Lists.newArrayList('+', '-', '&', '|', '!', '(', ')', '{', '}', '[', ']', '^', '~',
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "SPECIAL_CHARS" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)